src/main/java/org/stellar/base/AssetTypeNative.java

```diff
@@ -7,6 +7,16 @@
   public AssetTypeNative() {}
 
   @Override
+  public String getCode() {
```

Comment:
`getCode` and `getIssuer` are abstract methods of abstract `Asset` class. I think it would be better to create another abstract class `AssetTypeCreditAlphaNum` and move these methods there. So it will look like this:

```
AssetTypeCreditAlphaNum4  AssetTypeCreditAlphaNum12    AssetTypeNative
              ^                 ^                             ^
              |                 |                             |
              +                 +                             |
            AssetTypeCreditAlphaNum                           |
                       ^                                      |
                       |                                      |
                       +                                      |
                     Asset +----------------------------------+
```
Then `AssetTypeNative` won't have to implement these methods.